src/Microsoft.ML.Transforms/Text/NgramHashingTransformer.cs

```diff
@@ -353,7 +353,7 @@ private static IDataTransform Create(IHostEnvironment env, Options options, IDat
                         item.NgramLength ?? options.NgramLength,
                         item.SkipLength ?? options.SkipLength,
                         item.AllLengths ?? options.AllLengths,
-                        item.HashBits ?? options.HashBits,
+                        item.NumberOfBits ?? options.NumberOfBits,
                         item.Seed ?? options.Seed,
                         item.Ordered ?? options.Ordered,
```

Comment:
>Ordered [](start = 29, length = 7)

UseOrderedHashing

Reply:
Fixed in Iteration 9.

---
In reply to: [264917459](https://github.com/dotnet/machinelearning/pull/2898#discussion_r264917459) [](ancestors = 264917459)